sample-android/src/test/java/com/github/takahirom/roborazzi/sample/boxed/ContextDataTest.kt

```diff
@@ -150,13 +150,12 @@ class ContextDataTest {
   ) {
     val any = contextData[key]
     if (any is Double) {
-      // gson always parse number as double
       assert(any.toInt() == value.toInt()) {
         "Expected $value but got $any"
       }
       return
     }
-    assert(any == value) {
+    assert(any.toString() == value) {
```

Comment:
This part seems a bit unnatural.
But without `toString()`, the assertion fails.
What do you think?